reports.go

```diff
@@ -105,3 +681,72 @@ func (c *Client) CreateReport(reqPayload *CreateReportRequest) (*Report, error)
 
 	return createResp, nil
 }
+
+const getReport = "/v1/reports"
+
+// GetReport Retrieves a report
+func (c *Client) GetReport(reportID string) (*Report, error) {
+	rel, err := url.Parse(getReport)
+	if err != nil {
+		return nil, err
+	}
+
+	u := *c.BaseURL
+	u.Path = path.Join(u.Path, rel.String(), reportID)
+
+	req, err := http.NewRequest(http.MethodGet, u.String(), nil)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Add("Content-Type", "application/json")
+	req.SetBasicAuth(c.APIKey, "")
+
+	resp, err := c.client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.StatusCode == http.StatusNotFound {
+		return nil, nil
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, NewError([]int{
+			http.StatusCreated,
+		}, resp)
+	}
+	defer func() {
+		if resp.Body != nil {
+			resp.Body.Close()
+		}
+	}()
+
+	b, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	report := &Report{}
+	err = json.Unmarshal(b, report)
+	if err != nil {
+		return nil, err
+	}
+
+	report.GetSSNTrace(report.SSNTraceID, c)
```

Comment:
This doesn't do anything because you're not doing anything to the response. You're also not checking the error.

Reply:
yaeh this is the question i had . Where should i fill the response i got. that is why i left the return types empty. where are we going to use it. The report has the ssn trace ID only.  like that we got 6 to 7 sub reports right. where shall we fill them. should i add additional fields to Reports and store it there ?

Reply:
@chandradeepak actually, for this one, just create a function call `GetScreenings` and create a new struct, `CheckrScreenings` that will contain all the screenings. So when you call `GetScreenings` it will return the `CheckrScreenings` struct. Also, make sure this fetches reports concurrently.

Reply:
checkr.CheckrScreening linter was complaining it is stuttering. so i made it screenings. so it is going to be checkr.GetScreenings which will return a screening object. This is done concurrently . if any thing errors we continue to the next screening and return what we get. it is implemented in screenings.go